content/python/concepts/file-methods/terms/unlink/unlink.md

```diff
@@ -0,0 +1,36 @@
+---
+Title: '.unlink()'
+Description: 'Allows the user to delete a file path.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Methods'
+  - 'Files'
+  - 'Functions'
+  - 'Strings'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The Python `.unlink()` file method allows the user to delete a file path if it exists in the system or computer. This method is provided by the `os` module which must be imported beforehand. [`.remove()`](https://www.codecademy.com/resources/docs/python/file-methods/remove) is the alternative to delete a specific file.
+
+## Syntax
+
+```pseudo
+os.unlink("file_path")
+```
+
```

Comment:
What happens if the given `"file_path"` doesn't exist?

Reply:
Same as with the `remove` method but I can add the note here as well

Reply:
Adding a note in the next push~